vllm/model_executor/layers/quantization/input_quant_fp8.py

```diff
@@ -46,6 +49,7 @@ def __init__(self,
         self.group_shape = group_shape
         self.num_token_padding = num_token_padding
         self.column_major_scales = column_major_scales
+        self.use_ue8m0 = use_ue8m0
```

Comment:
Does `use_ue8m0` change the output/semantics of this op? If so, we should assert `use_ue8m0` is false/unset in the `_group_native` method as it doesn't do anything right now.

Reply:
It is only used in `forward_cuda()`, I'll put an assert in `forward_native()`